declarations.d.ts

```diff
@@ -282,7 +282,7 @@ declare module "webpack-sources" {
 		insert(pos: number, newValue: string, name?: string): void;
 
 		getName(): string;
-		original(): string;
+		original(): Source;
```

Comment:
@IronLu233 just remove other changes, keep only `HarmonyEvaluatedImportSpecifierDependency.js:106`

Reply:
OK. But the original definition is wrong.Change it in another PR is better.